.github/workflows/e2e-leg-2.yml

```
@@ -33,6 +36,10 @@ on:
         description: 'Name of the vertica server image'
         type: string
         required: false
+      vertica-deployment-method:
+        description: 'Back-end library(admintools|vclusterops)'
+        type: string
+        required: false
```

minor: I suggest we use less generic language and use a choice. Same comment in the other workflow file.
```suggestion
      vertica-deployment-method:
        description: 'Vertica's deployment method'
        type: choice
        required: false
        default: admintools
        options:
        - admintools
        - vclusterops
```